CollAction/Startup.cs

```diff
@@ -166,7 +166,12 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                      "categories",
                      new { controller = "Projects", action = "GetCategories" }
                  );
-                
+
+                routes.MapRoute("GetTileProjects",
```

Comment:
Yeah I would expect /Projects/Find just like we already have. The only difference would be the header that is passed in.